src/routes/+layout.svelte

```
@@ -39,7 +39,7 @@
 
 	import 'tippy.js/dist/tippy.css';
 
-	import { WEBUI_BASE_URL, WEBUI_HOSTNAME } from '$lib/constants';
+	import {TRIAL_USER_EMAIL, WEBUI_BASE_URL, WEBUI_HOSTNAME} from '$lib/constants';
```

ESLint

[error] 'WEBUI_HOSTNAME' is defined but never used.

(@typescript-eslint/no-unused-vars)